packages/core-data/src/entities.js

```diff
@@ -54,12 +79,19 @@ function* loadPostTypeEntities() {
 			kind: 'postType',
 			baseURL: '/wp/v2/' + postType.rest_base,
 			name,
+			label: postType.labels.singular_name,
 			transientEdits: {
 				blocks: true,
 				selectionStart: true,
 				selectionEnd: true,
 			},
 			mergedEdits: { meta: true },
+			getTitle( record ) {
```

Comment:
This is supposed to be a generic way to get a "title"/"caption"/"label" of any Entity Record. Notice that it still needs some specific behavior.